### ISSUE TITLE
Decide tech stack for v1.0

### ISSUE THREAD

**Issue by chiraganand** [nipfpmf/law-typesetting#2]
My initial thoughts are with Python Django. AngularJs seems a bit of overkill at this stage when the features required are simple.

My list:
**Front-end**: Django templates (with Jinja templating language)
**Backend**: Python Django
**Database**: SQLite (to begin with)

**Comment by chiraganand** [comment]
Other reason for going with Python is that it is widely accepted language for community projects and fairly easy to pick up for new contributors.

**Comment by srivatsshankar** [comment]
Agreed! It will help us get started quickly. We can worry about the front-end later and figure that out using some restful API.

We can even possibly develop modules for parsing XML statutes separately as well, since Python makes that super easy!

**Comment by chiraganand** [comment]
> Agreed! It will help us get started quickly. We can worry about the front-end later and figure that out using some restful API.

Django provides out of the box support for building REST API for its
apps so it should be easy.

> We can even possibly develop modules for parsing XML statutes separately as well, since Python makes that super easy!

Yes, let us think where XML parsing fits in. Ideally, a Django app
should be created for that and the parsing modules can be part of this
app.
